engine/order_manager.go

```diff
@@ -875,3 +973,46 @@ func (s *store) getFilteredOrders(f *order.Filter) ([]order.Detail, error) {
 	}
 	return os, nil
 }
+
+// getActiveOrders returns copy of the orders that are active
+func (s *store) getActiveOrders(f *order.Filter) ([]order.Detail, error) {
+	if f == nil {
+		return nil, errors.New("filter is nil")
+	}
+	s.m.RLock()
+	defer s.m.RUnlock()
+
+	var orders []order.Detail
+	switch {
+	case f == nil:
```

Comment:
The first line in this function blocks this first `switch` `case` from being hit. Please either remove the `f==nil` check of remove this case